delfin/common/constants.py

```diff
@@ -288,3 +285,11 @@ class ResourceSync(object):
     START = 100
     SUCCEED = 100
     FAILED = 101
+
+
+class Task(object):
+    DEFAULT_TASK_INTERVAL = 30
```

Comment:
move to conf

Reply:
Currently the value fetched from Constants for the completion of flow. Agree to move in conf file.
Can take it in upcoming PR of scheduler.  Need to add all other configuration for scheduler, and scheduled task.